app/services/edge_cache/service.rb

```diff
@@ -0,0 +1,115 @@
+module EdgeCache
+  class Service
```

Comment:
`EdgeCache::Service` in the `/services` file feels a little redundant to me and vague given this is just for busting caches. What about EdgeCacheBuster or something along those lines that is more specific about its function?

Reply:
@mstruve Yeah, it's a little weird, isn't it? I think your suggestion makes sense; it will line up nicely with the `CacheBuster` module too, which internally just calls to `EdgeCache::CacheBuster.bust`

Latest commit renames this :)